src/parser.ts

```diff
@@ -207,7 +207,7 @@ export interface Schema extends XtpTyped {
 
 export type XtpType =
   'integer' | 'string' | 'number' | 'boolean' | 'object' |
-  'array' | 'buffer' | 'object' | 'enum' | 'map';
+  'array' | 'buffer' | 'enum' | 'map';
```

Comment:
why remove `object`? this should still be a valid type.
something i just noticed btw, `map` is not a valid type.

Reply:
oh, i see it now 😆

Reply:
Is `map` not a valid type? https://docs.xtp.dylibso.com/docs/concepts/xtp-schema has not been updated, I'd assume it could be passed as a `ValueParameter` like any type other than typed objects and enums. Is a different value used for `type` when using a map as a `ValueParameter`?

Reply:
`map` removed in a141e93f